data/Act.csv

```diff
@@ -1,79 +1,79 @@
+A00046,P0018,AT01,T00062,,1976,1976,,,T00044,311,,2017-09-06,MG,2017-09-06,MG,,,,
+A00047,P0102,AT02,T00063,,1979,1979,,,T00044,537,,2017-09-06,MG,2017-09-06,MG,,,,
+A00048,P0018,AT01,T00064,,MIRA,,Yan Li and a friend (Mao Tou) were discussing some poems of Bei Dao,,T00044,312,,2017-09-06,MG,2017-09-06,MG,,,,
+A00049,P0102,AT02,T00067,,1979,1979,,,T00044,537,,2017-09-08,MG,2017-09-08,MG,,,,
+A00050,P0102,AT02,T00068,,1979,1979,,,T00044,537,,2017-09-08,MG,2017-09-08,MG,,,,
+A00051,P0102,AT02,T00069,,1979,1979,,,T00044,537,,2017-09-13,MG,2017-09-13,MG,,,,
+A00052,P0134,AT02,T00066,,1973,1973,copied from Jiang He,,T00045,291,,2017-09-13,MG,2017-09-13,MG,,,,
+A00053,P0134,AT02,T00065,,1973,1973,copied from Jiang He,,T00045,291,,2017-09-13,MG,2017-09-13,MG,,,,
+A00054,P0014,AT01,T00070,,,,Xu Xiao remembers it,,T00045,153-182 or 375-414,,2017-09-15,MG,2017-09-15,MG,,,,
+A00055,P0102,AT13,T00071,,1979,1979,This is the first poem of the author,,T00044,534,,2017-09-20,MG,2017-09-20,MG,,,,
+A00056,P0102,AT13,T00072,,1979,1979,The author was deeply influenced by Sylvia Plath in the beginning of his work as a poet,,T00044,536,,2017-09-20,MG,2017-09-22,MG,,,,
+A00057,P0144,AT11,T00073,,,,According to Wang Shiqiang's research this piece was one of the most influential yellow-covered books for the generation of pre-'misty' poets,,T00011,78,,2017-09-20,MG,2017-10-02,DP,,,,
+A00058,P0102,AT01,T00021,,1979,1979,The author felt the resonance of this essay about the Maoist period and the Soviet time when people focused more on spirit instead of material,,T00044,534,,2017-09-22,MG,2017-09-22,MG,,,,
+A00059,P0106,AT01,T00076,,1972,1972,The author introduced the categories of those internal published books. The year 1972 was overwhelming to those book lovers for they sensed there seemed to be a chance of opening  up.,,T00005,580,,2017-09-22,MG,2017-09-22,MG,,,,
+A00060,P0153,AT01,T00077,,1966,1976,he wrote 'research of the Dream of the Red Chamber' and was put on the hat of 牛鬼蛇神. In 1954 a campaign was launched to criticise his 'bourgeoise' interpretation of the book.,,T00044,372,,2017-09-22,MG,2017-09-26,MG,,,,
+A00061,P0029,AT01,T00077,,,,He read it before his university studies and mentions that this book was one of the most-beloved and widely-read among his book friends.,,T00029,116-119,,2017-09-22,MG,2017-09-22,MG,,,,
+A00062,P0154,AT01,T00077,,,,He wanted the book so bad and it was really hard to have one. But he finally got one from the brother of one of his classmate and read it secretly.,,T00044,390,,2017-09-22,MG,2017-09-22,MG,,,,
+A00063,P0155,AT01,T00077,,1964,1965?,,,T00045,46,,2017-09-22,MG,2017-09-22,MG,,,,
+A00064,P0156,AT01,T00078,,1966,,after reading lots of foreign and Chinese classics, he still found Lu Xun the best,,T00027,ONLINE SOURCE,,2017-09-24,CW,2017-10-02,DP,,,
```

Comment:
You columns are out of sync, avoid commas when inputting text

Reply:
Yes, I have seen this too, but when I open my Github Desktop now, the columns are in sync. Feels a bit irritating.

Reply:
It depends on how you display the files in this branch as tables. If it doesn't display properly on Github, it won't do so in 100 other contexts. Point remains ``commas = bad``